### PR TITLE
Triple template issue

### PR DESCRIPTION
**Pull request by ansharlubis**
Added the triple template example (issue #1038).

### CODE REVIEW

File: integration_tests/CMakeLists.txt [hunk 1]
```diff
@@ -620,6 +620,7 @@ RUN(NAME conv_complex2real LABELS gfortran llvm)
 RUN(NAME template_add LABELS llvm)
 RUN(NAME template_nested LABELS llvm)
 RUN(NAME template_travel LABELS llvm)
+RUN(NAME template_triple LABELs llvm)
```
Comment (certik) [inline integration_tests/CMakeLists.txt:623]:
```suggestion
RUN(NAME template_triple LABELS llvm)
```

**Comment by certik** [comment]
I rebased it to run all tests.

I think there are no parsing issues, right? So I would not test AST at all.

Just test ASR. And if this compiles and runs, you can possibly just run it, but I think we are not 100% sure that everything works even if the code executes, so testing ASR as you do is good.

So just remove the AST test and this should be good.

**Comment by ansharlubis** [comment]
> I rebased it to run all tests.
> 
> I think there are no parsing issues, right? So I would not test AST at all.
> 
> Just test ASR. And if this compiles and runs, you can possibly just run it, but I think we are not 100% sure that everything works even if the code executes, so testing ASR as you do is good.
> 
> So just remove the AST test and this should be good.

I've removed the AST test for the triple example added here.

**Comment by certik** [comment]
Looks good, thanks!

**Pull request merged**